app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,81 @@
+class ProjectsController < ApplicationController
+  responders :flash
+
+  # GET /projects
+  def index
+    @projects = policy_scope(Project).all.sort_by(&:id)
+    @project = Project.new
+    respond_with @projects
+  end
+
+  # GET /projects/1
+  def show
+    authorize project
+    respond_with project
+  end
+
+  # GET /projects/new
+  def new
+    skip_authorization
+
+    @project = Project.new
+    respond_with @project
+  end
+
+  # GET /projects/1/edit
+  def edit
+    authorize project
+    respond_with project
+  end
+
+  # POST /projects
+  def create
+    skip_authorization
+
+    project_id = params[:project][:id]
+
+    unless credential.accessible_project?(project_id)
+      head :forbidden
+      return
+    end
+
+    if Rails.env.development? || Rails.env.test?
```

Comment:
I'm not sure that environment logic like this belongs in the controller. The Effects module already has a FakePanoptes class that will be returned in dev/test envs for this purpose.

Reply:
I wasn't sure how to use the FakePanoptes class--I'm not sure whether it's appropriate to stub methods onto there the way that we do the FakeCredential class definition.

Reply:
`Effects.panoptes` already does the env check and returns the fake one if you're in dev/test. It returns nil, looks like, so you might have to allow/to/receive in the spec.

Reply:
Yeah, that's easy enough to do for the spec, but it presents a bit more of a challenge when I'm trying to develop locally!